apps/api/migrations/081_add_savings_goal_soft_delete.down.sql

```
@@ -0,0 +1,4 @@
+DROP INDEX IF EXISTS idx_savings_goals_deleted_at;
```

SQLFluff (4.2.2)

[error] DROP statement should use CONCURRENTLY to avoid locking the table.

(PG01)